homeassistant/components/comelit/light.py

```diff
@@ -0,0 +1,78 @@
+"""Support for lights."""
+from __future__ import annotations
+
+from typing import Any
+
+from aiocomelit import ComelitSerialBridgeObject
+from aiocomelit.const import LIGHT, LIGHT_OFF, LIGHT_ON
+
+from homeassistant.components.light import LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import ComelitSerialBridge
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Comelit lights."""
+
+    coordinator: ComelitSerialBridge = hass.data[DOMAIN][config_entry.entry_id]
+
+    # Last resort: config_entry.entry_id as unique_id as no serial number or mac are available
```

Comment:
It's OK to use the config entry id as unique ID base, but please adjust the parameter name to the constructor
```suggestion
    # Use config_entry.entry_id as base for unique_id because no serial number or mac is available
```